src/components/pages/home/index.tsx

```diff
@@ -53,7 +53,7 @@ const Home: FunctionComponent = () => {
   const portfolioBlog: any = find(homepageData, {id: 'portfolioBlog'})
   const topics: any = find(homepageData, {id: 'topics'})
   const swag: any = find(homepageData, {id: 'swag'})
-  const cms: any = find(homepageData, {id: 'cms'})
+  const ecommerce: any = find(homepageData, {id: 'ecommerce'})
```

Comment:
instead of specifying a topic, consider specifying the target placement

Reply:
oh right, that makes sense